Simplenote/src/main/java/com/automattic/simplenote/utils/PrefUtils.java

```diff
@@ -32,6 +32,9 @@
     // boolean, set on first launch
     public static final String PREF_FIRST_LAUNCH = "pref_key_first_launch";
 
+    // boolean, set on first launch
```

Comment:
Is this a copy/paste artifact?  Maybe it should be something like `// boolean, require account login` instead.